### ISSUE TITLE
Info on AfterParse

### ISSUE THREAD

**Issue by GoogleCodeExporter** [geocodezip/geoxml3#50]
```
What steps will reproduce the problem?
1.
2.
3.

What is the expected output? What do you see instead?
I want to parse a kmldocument by using the parse method;
What i want is not let the parser showing any info on the map;
i want use the AfterParse callback so that i can create my Markers  by myself; 
i thought it was enough to set "map:null" while instantiating the object 
geoXML3.parser but it seems mandatory hence  i am not able to suppress the 
markers generated autmatically by the parse method
Basically i am following the example "Use Case 3: Standalone KML Parser" but it 
 does not work; am i missin something

What version of the product are you using? On what operating system?


Please provide any additional information below.


```

Original issue reported on code.google.com by `mic68.sa...@gmail.com` on 4 Feb 2012 at 8:56
